dali/python/nvidia/dali/ops.py

```diff
@@ -144,7 +152,7 @@ def _docstring_prefix_from_inputs(op_name):
     # Signature
     ret = "__call__(" + schema.GetCallSignatureInputs() + ", **kwargs)\n"
     # __call__ docstring
-    ret += "\nOperator call to be used in `define_graph` step.\n"
+    ret += "\nOperator call to be used in the `define_graph` step.\n"
```

Comment:
```suggestion
    ret += "\nOperator call to be used in graph definition.\n"
```
Let's not mention `define_graph` by name, since it's not the only way to define the graph.

Reply:
Done